byzcoin/bcadmin/lib/config.go

```diff
@@ -30,6 +30,19 @@ type Config struct {
 	AdminIdentity darc.Identity
 }
 
+//BaseConfig is a smaller version of Config containing only the necessary material
```

Comment:
I think these structures are only used in the QR generator to tell the JSON encoder what to do. If so, then move them inside of the qr() function. (Types can be defined anywhere variables can, and have the same limited scope. Smaller scope means less for the reader of your code to think about. see https://research.swtch.com/names for a meditation on how the scope of something in a program effects the brainpower needed to process it.)

Also: their type name should be lower case, since they do not need to be exported.

Reply:
Fixed